.github/workflows/docs.yml

```diff
@@ -57,7 +57,11 @@ jobs:
       - name: Install custom OSMesa VTK variant
         run: |
           pip uninstall vtk -y
-          pip install https://github.com/pyvista/pyvista-wheels/raw/main/vtk_osmesa-9.2.5-cp311-cp311-linux_x86_64.whl
+          aria2c -x5 "https://github.com/pyvista/pyvista-wheels/raw/main/${VTK_OSMESA_WHEEL_NAME}"
```

Comment:
I've never heard of aria2c (but I also don't know devops stuff in general), so out of curiosity: why `aria2c` and not something more mainstream like `curl` or `wget`?

Reply:
`aria2c` downloads the online resource with multiple stream in parallel. `-x` parameter specifies:

>The maximum number of connections to one server for each download

Reply:
Please take a look at [aria2 homepage](https://aria2.github.io/index.html).

> aria2 is a lightweight multi-protocol & multi-source command-line download utility. It supports HTTP/HTTPS, FTP, SFTP, BitTorrent and Metalink. aria2 can be manipulated via built-in JSON-RPC and XML-RPC interfaces.

Reply:
Neat, thanks.